ServerlessLibraryLogicApp/LogicApp.json

```diff
@@ -0,0 +1,400 @@
+                    "path": "/dbs/@{encodeURIComponent('serverlesslibrary')}/colls/@{encodeURIComponent('contributions')}/docs"
+                  },
+                  "runAfter": {}
+                },
+                "SendApprovalNotificationEmail": {
+                  "type": "ApiConnection",
+                  "inputs": {
+                    "host": {
+                      "connection": {
+                        "name": "@parameters('$connections')['office365']['connectionId']"
+                      }
+                    },
+                    "method": "post",
+                    "body": {
+                      "To": "serverless-admins@microsoft.com",
+                      "Subject": "Request for sample \"@{body('ParseJSON')?['title']}\" was approved",
+                      "Body": "@triggerBody()?['MessageText']"
+                    },
+                    "path": "/Mail"
+                  },
+                  "runAfter": {
+                    "AddContributionToCosmosDB": [
+                      "Succeeded"
+                    ]
+                  }
+                }
+              },
+              "runAfter": {
+                "SendApprovalEmail": [
+                  "Succeeded"
+                ]
+              },
+              "else": {
+                "actions": {
+                  "SendRejectionNotificationEmail": {
+                    "type": "ApiConnection",
+                    "inputs": {
+                      "host": {
+                        "connection": {
+                          "name": "@parameters('$connections')['office365']['connectionId']"
+                        }
+                      },
+                      "method": "post",
+                      "body": {
+                        "To": "serverless-admins@microsoft.com",
+                        "Subject": "Request for sample \"@{body('ParseJSON')?['title']}\" was rejected!",
+                        "Body": "@triggerBody()?['MessageText']"
+                      },
+                      "path": "/Mail"
+                    },
+                    "runAfter": {
+                      "CopyRequestToRejectedQueue": [
+                        "Succeeded"
+                      ]
+                    }
+                  },
+                  "CopyRequestToRejectedQueue": {
+                    "type": "ApiConnection",
+                    "inputs": {
+                      "host": {
+                        "connection": {
+                          "name": "@parameters('$connections')['azurequeues']['connectionId']"
+                        }
+                      },
+                      "method": "post",
+                      "body": "@triggerBody()?['MessageText']",
+                      "path": "/@{encodeURIComponent('rejected-requests')}/messages"
+                    },
+                    "runAfter": {}
+                  }
+                }
+              }
+            },
+            "DeleteRequestFromQueue": {
```

Comment:
You can keep the contribution item in another queue, instead of deleting. This may help us in DR

Reply:
I am already doing it for rejected requests. do you mean for the success case as well?